api/app/schemas/weather_models.py

```diff
@@ -48,6 +48,7 @@ class WeatherModelPredictionValues(BaseModel):
     wind_speed: float = None
     wind_direction: float = None
     total_precipitation: float = None
```

Comment:
Can we please drop total_precipitation - I think this is just going to cause confusion on the front-end.